mautrix_telegram/example-config.yaml

```diff
@@ -99,19 +99,57 @@ metrics:
 telemetry:
     # Enable usage telemetry. Mandatory for on-premise installations.
     enabled: false
+
     # EMS instance identifier.
     # Set to "generate" to generate and save a new ID.
     # Alternatively, set the "MAUTRIX_TELEGRAM_LICENCE_PATH" environment variable to specify
     # a file to read the ID from, which is generated if it does not exist.
     # If neither instance_id nor a file path are set, a default file is used.
     instance_id: generate
-    # The EMS endpoint to submit telemetry entries to.
-    # If set to a blank string, no telemetry is sent, but is still gathered.
-    endpoint: https://ems.element.io/api/remote/telemetry
-    # How many times to retry sending telemetry if it fails.
-    retry_count: 3
-    # How long to wait between retries, in seconds.
-    retry_interval: 60
+
+    # Config for the Matrix room in which telemetry data will be stored.
+    matrix_destination:
```

Comment:
Not sure if needed, but the Synapse telemetry module also allows for configuration of the [sender localpart](https://gitlab.matrix.org/new-vector/modular/synapse-telemetry/-/blob/254d208f7a82919240659e790a794d404631c561/README.md#L71).

Reply:
Yes -- as an appservice, the only senders that could be used are ones the bridge has ownership of: the bridge bot, or a generated Telegram puppet user.

For simplicity, this PR uses the bridge bot for all telemetry event/room handling.

A possibility is to use a dedicated custom bot (independent of the bridge bot) for telemetry, which would require an update to the bridge's registration file (which could be automated).  Though that seems very much like a "nice-to-have" feature, so in the interest of time, I'll only implement it on request.